### PR TITLE
[Docs] Update all the metafiles

### PR DESCRIPTION
**Pull request by JingweiZhang12**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by ZCMax** [comment]
Please update centerpoint according to this PR https://github.com/open-mmlab/mmdetection3d/pull/1917, we update the centerpoint checkpoint link

File: configs/pointpillars/metafile.yml [hunk 1]
```diff
@@ -57,7 +57,29 @@ Models:
           NDS: 49.1
     Weights: https://download.openmmlab.com/mmdetection3d/v1.0.0_models/pointpillars/hv_pointpillars_secfpn_sbn-all_4x8_2x_nus-3d/hv_pointpillars_secfpn_sbn-all_4x8_2x_nus-3d_20210826_225857-f19d00a3.pth
 
-  - Name: hv_pointpillars_fpn_sbn-all_4x8_2x_nus-3d
+  - Name: hv_pointpillars_secfpn_sbn-all_fp16_2x8_2x_nus-3d
+    In Collection: PointPillars
+    Config: configs/pointpillars/hv_pointpillars_secfpn_sbn-all_fp16_2x8_2x_nus-3d.py
```
Comment (ZCMax) [inline configs/pointpillars/metafile.yml:62]: check the config name

File: configs/pointpillars/metafile.yml [hunk 2]
```diff
@@ -57,7 +57,29 @@ Models:
           NDS: 49.1
     Weights: https://download.openmmlab.com/mmdetection3d/v1.0.0_models/pointpillars/hv_pointpillars_secfpn_sbn-all_4x8_2x_nus-3d/hv_pointpillars_secfpn_sbn-all_4x8_2x_nus-3d_20210826_225857-f19d00a3.pth
 
-  - Name: hv_pointpillars_fpn_sbn-all_4x8_2x_nus-3d
+  - Name: hv_pointpillars_secfpn_sbn-all_fp16_2x8_2x_nus-3d
```
Comment (ZCMax) [inline configs/pointpillars/metafile.yml:60]: Why do we have this config in dev-1.x

File: configs/centerpoint/metafile.yml [hunk 3]
```diff
@@ -16,80 +16,80 @@ Collections:
       Version: v0.6.0
 
 Models:
-  - Name: centerpoint_01voxel_second_secfpn_circlenms_4x8_cyclic_20e_nus
+  - Name: /centerpoint_voxel01_second_secfpn_head-circlenms_8xb4-cyclic-20e_nus-3d
```
Comment (VVsssssk) [inline configs/centerpoint/metafile.yml:19]: /centerpoint_voxel01_second_secfpn_head-circlenms_8xb4-cyclic-20e_nus-3d -> centerpoint_voxel01_second_secfpn_head-circlenms_8xb4-cyclic-20e_nus-3d

File: configs/parta2/metafile.yml [hunk 4]
```diff
@@ -16,7 +16,7 @@ Collections:
       Version: v0.5.0
 
 Models:
-  - Name: hv_PartA2_secfpn_2x8_cyclic_80e_kitti-3d-3class
+  - Name: PartA2_hv_secfpn_8xb2-cyclic-80e_kitti-3d-3class
```
Comment (VVsssssk) [inline configs/parta2/metafile.yml:19]: Rename all 'PartA2' -> 'parta2' in config fold.

**Review by VVsssssk — APPROVED** [review]

**Review by ZCMax — APPROVED** [review]

**Comment by JingweiZhang12** [comment]
Ready to be merged.

**Pull request merged**